test/Microsoft.AspNetCore.Razor.Tools.Test/ServerLifecycleTest.cs

```diff
@@ -116,7 +116,9 @@ public void ServerStartup_SuccessfullyAcquiredMutex()
         /// A shutdown request should not abort an existing compilation.  It should be allowed to run to 
         /// completion.
         /// </summary>
-        [Fact(Skip = "Skipping temporarily on non-windows. https://github.com/aspnet/Razor/issues/1991")]
+        // Skipping temporarily on non-windows. https://github.com/aspnet/Razor/issues/1991
+        [ConditionalFact]
```

Comment:
I disabled these to debug a CI flakiness issue a while ago but it turned out these were not the cause. I forgot to re-enable these.